server/Makefile

```diff
@@ -50,6 +50,7 @@ binlinks = \
 	pbench-satellite-cleanup\
 	pbench-satellite-state-change\
 	pbench-server-prep-shim-002\
```

Comment:
Doesn't `pbench-server-put-shim` *replace* `pbench-server-prep-shim-002`???

No, don't worry about that too much right now. I think we need to decide whether the tip of 0.69 will really become the minimalized "shim server" we're envisioning for our production server migration, or whether we want to support hypothetical other deployments of a "real" standalone 0.69-10 server. I think we probably want the shim-only, and truly tie off 0.69 server development, and I'm not sure that's entirely committed.

Reply:
yeah, I was also not sure about that. So, I didn't make changes to existing files.